### PR TITLE
Translate design systems for developers to zh-CN

### PR DESCRIPTION
**Pull request by XiKun-TW**
Proofreading from native speakers wanted. Review and comment on this PR 🙏.

### CODE REVIEW

File: content/design-systems-for-developers/react/zh-CN/document.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+我们的文档必须通过以下几点来克服创建和维护文档的固有痛点：
+
+- **🔄 时刻保持最新版本** 通过使用最新生产环境上的代码
+- **✍️ 易于编写** 使用例如：Markdown 这样熟悉的写作工具
+- **⚡️ 减少维护时间** 这样团队可以更专于写作
+- **📐 提供样板** 这样开发人员不需要重写一些通用的模板
+- **🎨 提供可定制** 来适用于极其复杂的用例和组件
+
+作为 Storybook 的用户，我们具有领先优势，因为组件的各个变体已经被记录为 stories（一种文档形式）。一个 story 展示了组件不在不同输入下应该如何工作， story 利于编写，且因其引用了生产环境的组件，所以可以自动更新。并且，我们可以使用上一章中的工具对 story 进行回归测试！
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/document.md]:
```suggestion
作为 Storybook 的用户，我们具有领先优势，因为组件的各个变体已经被记录为 stories（一种文档形式）。一个 story 展示了组件在不同输入下应该如何工作， story 利于编写，且因其引用了生产环境的组件，所以可以自动更新。并且，我们可以使用上一章中的工具对 story 进行回归测试！
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+```
+
+并且使用 Auto 工具在 GitHub 上创建一个 release：
+
+```bash
+git push --follow-tags origin master
+yarn auto release
+```
+
+赞! 我们已经成功的在 npm 中发布了我们的软件包并且在 GitHub 上创建了一个 release。 (祝好运!).
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/distribute.md]:
```suggestion
赞! 我们已经成功的在 npm 中发布了我们的软件包并且在 GitHub 上创建了一个 release（祝好运!）。
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+yarn storybook
+```
+
+您应该可以看到 Storybook 中包含了应用程序使用的简单组件的 stories：
+
+![Initial storybook for example app](/design-systems-for-developers/example-app-starting-storybook.png)
+
+<h4>集成设计系统</h4>
+
+将您您发布的设计系统添加为依赖项：
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/distribute.md]:
```suggestion
将您发布的设计系统添加为依赖项：
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+添加您的修改并提交.
+
+```bash
+git commit -am "Added AvatarList and stories"
+```
+
+## 文档
+
+感谢 Storybook Docs 插件，我们只需要花一点点时间便可以可自定义文档内容。它可以帮助其他人通过浏览 Storybook 的 Docs 页签去学习如何使用 AvatarList。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
感谢 Storybook Docs 插件，我们只需要花一点点时间便可以得到内容可自定义的文档。它可以帮助其他人通过浏览 Storybook 的 Docs 页签去学习如何使用 AvatarList。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+和团队的成员达成共识往往可能是徒劳的，因为他们可能参考了过时的代码，或者没有开发环境，或者将反馈散布在不同的工具上。通过在线的 Storybook 让审查变得像分享链接一样简单。
+
+## 测试
+
+我们的测试套件会在每次提交之后在后台运行。AvatarList 是一个简单的显示组件，所以不需要单元测试。但是如果我们查看提交请求检查，我们会发现视觉测试工具 Chromatic 已经识别出需要被审查的变化。
+
+![Chromatic changes on the GitHub PR check](/design-systems-for-developers/github-pr-checks-chromatic-changes.png)
+
+因为 AvatarList 是一个新的组件且未被视觉测试覆盖到，我们则需要为每个 story 去添加新的基准。在 Chromatic 中接受 “新到 stories” 去扩大我们的视觉测试覆盖率。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
因为 AvatarList 是一个新的组件且未被视觉测试覆盖到，我们则需要为每个 story 去添加新的基准。在 Chromatic 中接受 “新 stories” 去扩大我们的视觉测试覆盖率。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+当您操作完成后，Chromatic 的检查便会通过。
+
+![Chromatic changes to the AvatarList stories accepted](/design-systems-for-developers/chromatic-avatarlist-changes-accepted.png)
+
+依次更新 GitHub 中的合并提交检查。
+
+![Chromatic changes accepted on the GitHub PR check](/design-systems-for-developers/github-pr-checks-chromatic-changes-accepted.png)
+
+测试被成功的更新了。在以后，回溯将很难偷偷摸摸的就被加入到设计系统中。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
测试被成功的更新了。在以后，回溯将很难偷偷摸摸地就被加入到设计系统中。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+## 发布
+
+我们有一个开放的 pull request 去把 AvatarList 添加到设计系统中。我们已经编写了相关的 stories，测试也已经通过并且已经有相关的文档。最后，我们已经准备好使用 Auto 和 npm 来打包设计系统。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
我们有一个进行的 pull request 去把 AvatarList 添加到设计系统中。我们已经编写了相关的 stories，测试已经通过并且也有相关的文档。最后，我们准备好使用 Auto 和 npm 来打包设计系统。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+给 pull request 添加 `minor` 标签，这可以告诉 Auto 此次更新是一个小版本的更新。
+
+![GitHub PR with labels](/design-systems-for-developers/github-pr-labelled.png)
+
+现在合并您的 pull request 到主分支，您的软件包已经在上传了，您可以在 npm 网站中打开您的软件包并稍等几分钟去查看新的版本。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
现在合并您的 pull request 到主分支，您可以在 npm 网站中打开您的软件包并稍等几分钟去查看新的版本。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+
+![Package published on npm](/design-systems-for-developers/npm-published-package.png)
+
+成功了! 您的设计系统包已经成功上传。您不需要任何和命令行操作或使用繁杂的 npm。在实例中更新您的 `learnstorybook-design-system` 依赖项，然后开始使用 AvatarList。
+
+## 开始您的旅程
+
+_针对开发人员的设计系统_ 着重介绍了专业团队使用的端到端的工作流程，帮助您能在开发自己的设计系统时先行一步。伴随着您的设计系统的发展、重编，您可以扩展这些工具来满足您团队的需求。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
_针对开发人员的设计系统_ 着重介绍了专业团队使用的端到端的工作流程，帮助您能在开发自己的设计系统时先行一步。伴随着您的设计系统的发展、整理，您可以扩展这些工具来满足您团队的需求。
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+哇！如您所见，一个组件包含很多的状态。将每个组件的状态乘以在设计系统中组件的数量您就会发现持续关注每个组件是一项不可能完成的任务。实际上，手工审查每个元素的每一个状态不是长久之道，尤其当设计系统越来越庞大。
+
+**现在**我们更需要设置自动测试来节约**以后**的工作时间。
+
+## 测前须知
+
+在[之前的文章](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07) 我对 4 个涉及到专业的 Storybook 工作流程的前端团队做过调查。他们认为开发 stories 是一个最佳实践，它让测试变得容易和全面。
+
+**清楚的将组件支持的状态**表达为 stories 可以阐明不同的输入组合所对应的组件状态，尽可能去忽略不支持的状态来降低干扰。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/test.md]:
```suggestion
**清楚地将组件支持的状态**表达为 stories 可以阐明不同的输入组合所对应的组件状态，尽可能去忽略不支持的状态来降低干扰。
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+
+  ReactDOM.unmountComponentAtNode(div);
+});
+```
+
+我们可以通过执行 `yarn test` 来运行上述测试
+
+![Running a single Jest test](/design-systems-for-developers/jest-test.png)
+
+之前我们已经配置了我们的 Circel config.js 在每一次提交之后运行 `yarn test`。现在我们这条单元测试便可从中获益。在之后的反复修改组件时，我们也会对强大的 Link 组件保持信心。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/test.md]:
```suggestion
之前我们已经配置了我们的 Circle config.js 在每一次提交之后运行 `yarn test`。现在我们这条单元测试便可从中获益。在之后的反复修改组件时，我们也会对强大的 Link 组件保持信心。
```

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+UI 开发是一项团队合作的工作，我们需要开发人员、设计师和其他参与者保持一致的认知。为了更快的发布产品，我们应该在开发过程中及时引入其他的干系人，让他们及早了解到我们正在开发的 UI 组件。
+
+#### 用测试规避 UI 错误
+
+设计系统既是唯一的“真理之源”，也是 UI 组件唯一的 “错误之源”。设计系统中一个小小的 UI 错误，可能会像滚雪球一样导致整个产品出现问题。我们将自动执行测试来帮助您规避不可避免的错误，并让您能放心的交付稳定、无障碍访问的 UI 组件。
+
+#### 提早交付文档
+
+在设计系统中，文档是必不可少的，但是开发人员往往将创建文档作为最低的优先级。我们通过 UI 组件将会为您提供自动生成最小可行的文档，极大的降低了您开发文档的难度，您也可以自行修改自动生成的文档。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]:
```suggestion
在设计系统中，文档是必不可少的，但是开发人员往往将创建文档作为最低的优先级。我们通过 UI 组件将会为您提供自动生成最小可行的文档，极大地降低了您开发文档的难度，您也可以自行修改自动生成的文档。
```

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+任何一个设计系统都是由组件组合而成的。我们把 Storybook 作为工作台来单独构建组件为其消费者提供服务。所以我们需要集成一些插件（如： Actions，Source，Knobs）来帮助我们节省时间和提升组件的耐用性。
+
+#### 通过审核的方式收集反馈并达成共识
+
+UI 开发是一项团队合作的工作，我们需要开发人员、设计师和其他参与者保持一致的认知。为了更快的发布产品，我们应该在开发过程中及时引入其他的干系人，让他们及早了解到我们正在开发的 UI 组件。
+
+#### 用测试规避 UI 错误
+
+设计系统既是唯一的“真理之源”，也是 UI 组件唯一的 “错误之源”。设计系统中一个小小的 UI 错误，可能会像滚雪球一样导致整个产品出现问题。我们将自动执行测试来帮助您规避不可避免的错误，并让您能放心的交付稳定、无障碍访问的 UI 组件。
```
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]:
```suggestion
设计系统既是唯一的“真理之源”，也是 UI 组件唯一的 “错误之源”。设计系统中一个小小的 UI 错误，可能会像滚雪球一样导致整个产品出现问题。我们将自动执行测试来帮助您规避不可避免的错误，并让您能放心地交付稳定、无障碍访问的 UI 组件。
```

File: content/design-systems-for-developers/react/zh-CN/document.md [hunk 14]
```diff
@@ -0,0 +1,400 @@
...
+
+- **🔄 时刻保持最新版本** 通过使用最新生产环境上的代码
+- **✍️ 易于编写** 使用例如：Markdown 这样熟悉的写作工具
+- **⚡️ 减少维护时间** 这样团队可以更专于写作
+- **📐 提供样板** 这样开发人员不需要重写一些通用的模板
+- **🎨 提供可定制** 来适用于极其复杂的用例和组件
+
+作为 Storybook 的用户，我们具有领先优势，因为组件的各个变体已经被记录为 stories（一种文档形式）。一个 story 展示了组件在不同输入下应该如何工作， story 利于编写，且因其引用了生产环境的组件，所以可以自动更新。并且，我们可以使用上一章中的工具对 story 进行回归测试！
+
+> 当您编写 story 时您将免费获得组件的属性文档和使用用例 – Justin Bennett, Engineer at Artsy
```
[the remark this answers is not in the document]
Comment (XiKun-TW) [reply content/design-systems-for-developers/react/zh-CN/document.md]: The en version is `for free` I think it's better to translate it to `免费`. It also make sense on current context.

**Comment by XiKun-TW** [comment]
Hi @plrthink , I made some changes based on your suggestions. Except `for free` (open for others opinions), the others are very helpful. Thanks !

**Comment by plrthink** [comment]
My pleasure. I'm happy to get involved.

File: content/design-systems-for-developers/react/zh-CN/document.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (plrthink) [reply content/design-systems-for-developers/react/zh-CN/document.md]: Well, I think '免费的' emphasizes on the finance and the words want to express the low effort on generating the docs, like the word 'bonus'.

But I'm open on this.

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+
+本指南将带领大家一步步的去揭示：如何通过自动化工具和详细的工作流去规模化的构建一个设计系统。 我们将逐步从现有的组件库组装一个设计系统，然后设置它的核心服务、库和工作流程。
+
+![Design system overview](/design-systems-for-developers/design-system-overview.jpg)
+
+## 设计系统究竟是什么呢？
+
+在此之前我们先想想：可重复使用的用户界面这个概念并不新鲜。 样式指南、UI 套件和可共享的组件已经至少存在数十年了。如今设计人员和开发人员共同协作去构建整个 UI 组件。如同乐高积木一样， UI 组件封装了各类用户界面的视觉和功能模块。
+
+目前用户界面由数百个模块化的 UI 组件组装而成， 这些 UI 组件经过重新排列来展现出不同的用户体验。
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 这些 UI 组件经过重新排列来展现出不同的用户体验。
->这些 UI 组件经过重新排列来提供不同的用户体验。

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+
+设计系统包含可重用的 UI 组件，这些组件帮助开发团队在各个项目中构造复杂、稳定和无障碍访问的用户界面。由于开发人员和设计人员都在为 UI 组件作出贡献，因此设计系统作为学科之间的桥梁，成为不同组织间通用组件的 “真理之源”。
+
+![Design systems bridge design and development](/design-systems-for-developers/design-system-context.jpg)
+
+设计师经常会谈论到在其使用的工具内部构建设计系统。完整的设计系统应包含 assets （Sketch、 Figma 等）、整体设计原则、设计结构、设计方案等。有大量面向设计师的指南，深入探讨了这些主题，因此我们不在这里赘述。
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 有大量面向设计师的指南，深入探讨了这些主题，因此我们不在这里赘述。
->有大量面向设计师的指南已经深入探讨过这些主题，因此我们不在这里赘述。

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+- 🎨 设计变量： 特定的样式变量，例如品牌的颜色和间距
+- 📕 网站文档：用法说明、示例和注意事项等
+
+构建好的组件通过包管理工具进行打包、版本化并在各个应用程序中使用。
+
+## 您需要一个设计系统吗?
+
+尽管对设计系统进行了大量的宣传，但是设计系统并不适用于所有场景。如果您和一个体量适中的团队一起开发一个应用，那么您最好将可复用组件放在一个目录下进行管理而不是动用大量人力去构建一个设计系统。对于小的项目而言，设计系统的维护、集成和工具使用的成本可能会远超于它为您带来的收益。
+
+设计系统适用于在各个项目之间共享组件的场景，如果您发现您自己复制粘贴了相同的 UI 组件在不同的应用程序或者团队之间，那么这个指南是非常适合您的。
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 如果您发现您自己复制粘贴了相同的 UI 组件在不同的应用程序或者团队之间
->如果您发现您自己在不同的应用程序或者团队之间复制粘贴了相同的 UI 组件

**Comment by domyen** [comment]
🙏 Thank you for contributing @plrthink @PhonyLou @XiKun-TW! I'm super excited for this.

**Comment by XiKun-TW** [comment]
Hi @PhonyLou , Thanks for your comment. It's very helpful.

File: content/design-systems-for-developers/react/zh-CN/conclusion.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+
+#### Storybook 是否与设计工具集成？
+
+是到！Storybook 社区创建了易于设计工具集成的插件。例如： InVision 的 [Design System Manager](https://www.invisionapp.com/design-system-manager) 与 Storybook 集成在一起来展示 InVision 应用程序的 stories。当然还有社区为 [design tokens](https://github.com/UX-and-I/storybook-design-token)、[Sketch](https://github.com/chrisvxd/story2sketch) 和 [Figma](https://github.com/pocka/storybook-addon-designs) 创建的插件。
+
+![Design tool integrations](/design-systems-for-developers/storybook-integrations-design.jpg)
+
+#### 如果只有一个应用程序是否需要设计系统？
+
+不，这可能会引发创建和维护的成本。在规模的设计生产中，一个设计系统节省的时间可能会小于创建它而耗费的时间。
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/conclusion.md]: 不，这可能会引发创建和维护的成本。在规模的设计生产中，一个设计系统节省的时间可能会小于创建它而耗费的时间。
->
不，这会带来开发和维护上的机会成本。在小规模应用上，使用设计系统所需的工作量代价比节省出来的时间更大。

File: content/design-systems-for-developers/react/zh-CN/introduction.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+- 📚 [Storybook](http://storybook.js.org) 用于开发组件并生成文档
+- ⚛️ [React](https://reactjs.org/) 用于构建以组件为中心的 UI （通过 create-react-app）
+- 💅 [Styled-components](https://www.styled-components.com/) 用于限制组件样式的作用域范围
+- ✨ [Prettier](https://prettier.io/) 用于自动格式化代码
+
+#### 维护设计系统
+
+- 🚥 [CircleCI](https://circleci.com/) 用于持续集成
+- 📐 [ESLint](https://eslint.org/) 用于 Javascript 语法检查
+- ✅ [Chromatic](https://chromaticqa.com) 用于捕获组件中的视觉错误 (由 Storybook 维护人员执行)
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/introduction.md:61]:
```suggestion
- ✅ [Chromatic](https://www.chromatic.com/) 用于捕获组件中的视觉错误 (由 Storybook 维护人员执行)
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 20]
```diff
@@ -0,0 +1,257 @@
...
+
+设计系统包含了可展示的 UI 组件，这些组件基本上都是可视的。视觉测试验证了渲染后的 UI 组件的视觉效果。
+
+视觉测试会在一个保持一致的浏览器环境下对每个 UI 组件截图。新的截图将会自动与之前的基准截图进行对比，如果在视觉上有任何不同，您将会被告知。
+
+![Visual test components](/design-systems-for-developers/component-visual-testing.gif)
+
+如果您在创建现代化 UI，视觉测试可帮助您的前端开发团队节省人工审查的时间，也可以避免昂贵的 UI 回归测试。我们将使用 Storybook 维护者提供的工业级服务 Chromatic 来演示视觉测试。
+
+首先，通过您的 Github 在 [ChromaticQA.com](https://chromaticqa.com) 上注册一个账号
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:42]:
```suggestion
首先，通过您的 Github 在 [Chromatic.com](https://www.chromatic.com/) 上注册一个账号
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 21]
```diff
@@ -0,0 +1,257 @@
...
+
+![Signing up at Chromatic](/design-systems-for-developers/chromatic-signup.png)
+
+从那里选择您目前设计系统的 Git 仓库，此时会把访问权限同步到后台以便于检查您每次的提交请求。
+
+![Creating a project at Chromatic](/design-systems-for-developers/chromatic-create-project.png)
+
+通过 npm 安装 [storybook-chromatic](https://www.npmjs.com/package/storybook-chromatic)
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:50]:
```suggestion
通过 npm 安装 [chromatic](https://www.npmjs.com/package/chromatic)
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 22]
```diff
@@ -0,0 +1,257 @@
...
+
+```bash
+yarn add --dev storybook-chromatic
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:53]:
```suggestion
yarn add --dev chromatic
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 23]
```diff
@@ -0,0 +1,257 @@
...
+```
+
+打开您的命令行并跳转到 `design-system` 目录。然后运行您的第一次测试来生成您视觉测试的基准截图。(您将需要使用在 Chromatic 网站中提供的应用程序代码)
+
+```bash
+yarn chromatic test --app-code=<app-code>
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:59]:
```suggestion
npx chromatic --project-token=<project-token>
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 24]
```diff
@@ -0,0 +1,257 @@
...
+    // ...
+  },
+};
+// ...
+```
+
+在此运行测试命令
+
+```bash
+yarn chromatic test --app-code=<app-code>
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:81]:
```suggestion
npx chromatic --project-token=<project-token>
```

File: content/design-systems-for-developers/react/zh-CN/test.md [hunk 25]
```diff
@@ -0,0 +1,257 @@
...
+
+      - run: yarn install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+      - run: yarn test
+      - run: yarn chromatic test --app-code=<app-code> --exit-zero-on-changes
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:117]:
```suggestion
      - run: yarn chromatic --project-token=<project-token> --exit-zero-on-changes
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md [hunk 26]
```diff
@@ -0,0 +1,391 @@
...
+  }
+}
+```
+
+现在当我们执行命令 `yarn release`, 我们将以自动化的方式逐步执行上述所有的步骤（自动生成变更日志除外）。我们需要在我们的 circle 配置文件中添加如下命令来保证自动发布所有被合并到主分支的提交：
+
+```
+# ...
+- run: yarn test
+- run: yarn chromatic test -a 2wix88i1ziu
```
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/distribute.md:249]:
```suggestion
- run: yarn chromatic --project-token=2wix88i1ziu
```

**Comment by XiKun-TW** [comment]
Thanks @ghengeveld, I already accept the all commits.

**Comment by jonniebigodes** [comment]
@plrthink @PhonyLou @XiKun-TW this is a friendly reminder to see how this is going? Also i don't know if you've checked but the design systems tutorial was updated recently. 

If you're able can you make the changes so that the tutorial is updated to the latest version and we get a accurate translation?

Stay safe all of you

**Comment by PhonyLou** [comment]
Hey @jonniebigodes , I cannot make the change merged. Maybe @XiKun-TW can help.

Hope you guys all good and safe~

**Comment by XiKun-TW** [comment]
Hi @jonniebigodes , I'll check all the changes, thanks for reminding. :-D

**Comment by jonniebigodes** [comment]
@XiKun-TW no need to thank. Glad that this will get updated soon and we'll get it merged once it's all fine tuned. Sorry for not reminding you earlier, but these have been a bit of a hectic weeks. 

@PhonyLou once @XiKun-TW addresses the feedback i would like for you to take another look and proofread it, so that we can get it merged.

And it goes without saying thank you very much for taking the time to translate this

Stay safe both of you.

**Comment by XiKun-TW** [comment]
Hi @jonniebigodes @PhonyLou , the latest translate is ready for proofread again. Pls help review, Thanks.

**Comment by plrthink** [comment]
I just reviewed the changes made these days, and I think it looks good to me.

**Comment by jonniebigodes** [comment]
@XiKun-TW i'll take a look shortly and see if we're good to go, thank you @plrthink for taking the time to take a look at the translation as well. 

I'll let you know before the merge is done.

File: content/design-systems-for-developers/react/zh-CN/review.md [hunk 27]
```diff
@@ -74,46 +35,107 @@ jobs:
...
+![Chromatic in the command line](/design-systems-for-developers/chromatic-manual-storybook-console-log.png)
 
-![Succeeded running our first build in Netlify](/design-systems-for-developers/netlify-deployed.png)
+复制链接提供的地址，并在新的浏览器窗口打开该地址来浏览您发布的 Storybook。 您将看到在线的 Storybook 和您本地开发环境下的一模一样。
 
-点击链接即可查看您发布成功的 Storybook。您会发现本地的 Storybook 和在线的版本一模一样。这样您的团队便可以像您查看本地的 UI 组件一样轻松的查看实际渲染出来的 UI 组件。
+![Storybook built with Chromatic](/design-systems-for-developers/chromatic-published-storybook.png)
 
-![Viewing our first build in Netlify](/design-systems-for-developers/netlify-deployed-site.png)
+这便可以让您的团队非常轻而易举的能够看到和您本地一模一样的 UI 组件。
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/review.md:75]: 这便可以让您的团队非常轻而易举的能够看到和您本地一模一样的 UI 组件。
->
这便可以让您的团队轻而易举地看到和您本地一模一样的 UI 组件。

File: content/design-systems-for-developers/react/zh-CN/review.md [hunk 28]
```diff
@@ -74,46 +35,107 @@ jobs:
...
-storybook-static
+
+并对变更进行提交:
+
+```bash
+git commit -m "Storybook deployment with GitHub action"
 ```
 
-然后提交您的改动
+最后通过如下命令将您对提交上传到远端代码仓库:
```
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/review.md:128]: 最后通过如下命令将您对提交上传到远端代码仓库:
->
最后通过如下命令将您的提交上传到远端代码仓库：

**Comment by PhonyLou** [comment]
For others, I think they are good enough. Thanks.

**Comment by XiKun-TW** [comment]
> For others, I think they are good enough. Thanks.

Thanks for review! @jonniebigodes, Could you pls help confirm wether if it's ok to merge to master?

**Comment by jonniebigodes** [comment]
@XiKun-TW sorry for not addressing this earlier, but it's been some hectic months with the release of Storybook 6.0 and updating the tutorials.

I wanted to know if you're ok with updating this to the most recent version of the tutorial, now that #380 was merged? I was thinking that as groundwork is already done, it would be a waste to merge this now and if you wanted to make a follow up pull request with the new updates.

Let me know if you're ok with updating the tutorial so that we can work from there and offer the best/most recent translation to the community.

Stay safe.

**Comment by XiKun-TW** [comment]
> @XiKun-TW sorry for not addressing this earlier, but it's been some hectic months with the release of Storybook 6.0 and updating the tutorials.
> 
> I wanted to know if you're ok with updating this to the most recent version of the tutorial, now that #380 was merged? I was thinking that as groundwork is already done, it would be a waste to merge this now and if you wanted to make a follow up pull request with the new updates.
> 
> Let me know if you're ok with updating the tutorial so that we can work from there and offer the best/most recent translation to the community.
> 
> Stay safe.

Sure, I'll update translate to recent version.

File: content/design-systems-for-developers/react/zh-CN/document.md [hunk 29]
```diff
@@ -0,0 +1,412 @@
...
+    component: Avatar,
+    componentSubtitle: 'Displays an image that represents a user or organization',
+  },
+};
+```
+
+然后给 Avatar 添加 JSDoc（在文件 `src/components/Avatar.js` 中），它为该组件提供了相应的描述。
+
+```javascript
+// src/components/Avatar.js
```
Comment (MichaelArestad) [inline content/design-systems-for-developers/react/zh-CN/document.md:95]: Hey! I just noticed this reference to a `components` directory and created [a PR to remove it](https://github.com/chromaui/learnstorybook.com/pull/508). I would remove it here and a handful of times where it appears in code further down in this document.

**Review by ghengeveld — APPROVED** [review]
Time to get this thing merged.

**Pull request merged**